modules/serverless-rn-1-13-0.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-13-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.13.0
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.19.0.
+* {ServerlessProductName} uses Knative Eventing 0.19.2.
+* {ServerlessProductName} now uses Kourier 0.19.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.19.1.
+* {ServerlessProductName} now uses Knative Kafka 0.19.1.
+* A `DomainMapping` custom resource has been added to {ServerlessProductName} to enable users to map a custom domain name to a Knative Service. For more information, see the https://knative.dev/docs/serving/creating-domain-mappings/[Knative documentation].
+
```

Comment:
Actually, we also need to mention the removal of v1alpha1 and v1beta1 versions of Knative resources. I'll try to provide more details shortly.

Reply:
The [upstream release notes for Serving 0.18.4](https://github.com/knative/serving/releases/tag/v0.18.3) mention this:
```
Serving v1alpha1 & v1beta1 will EOL in our next release (v0.19)
This applies to the resources: Service, Route, Revision, Configuration
You will want to migrate your storage version for these resources to v1 using our post-install job
```
The upstream Serving 0.19 release notes then mention this:
```
Drop serving v1alpha1 and v1beta1 APIs
```
We now include 0.19 and the APIs were dropped. Maybe we want to mention it in the current release notes?